docs/querying/sql-functions.md

```diff
@@ -1092,6 +1140,14 @@ Returns a slice of the array from the zero-based start and end indexes.
 
 Joins all elements of the array together by the given delimiter.
 
+## NTILE
+
+`NTILE(tiles)`
+
+**Function type:** [Window](sql-window-functions.md#window-function-reference)
+
+Divides the rows within a window as evenly as possible into the number of tiles, also called buckets, and returns the value of the tile that the row falls into.
```

Comment:
I'm a bit confused by this. Say A window size is 10 and tiles is 2. This wording indicates that it divides the window into 2 buckets but it ideally should mean divide the 10 into 5 buckets with 2 items in each. @kgyrtkirk can you please chime in